### PR TITLE
493 build mobile beta nav

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Implement mobile version of beta navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check the navigation matches the mobile design in the issue

## Issue / Card

Fixes [#493](https://github.com/ubuntudesign/web-squad/issues/493)

### CODE REVIEW

File: static/sass/_global-settings.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 $breakpoint-navigation-threshold:    1030px;
+$breakpoint-mobile-small: 520px;
```
[the remark this answers is not in the document]
Comment (steverydz) [reply static/sass/_global-settings.scss]: Sure, I'll update it

File: static/sass/_global-settings.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 $breakpoint-navigation-threshold:    1030px;
```
[the remark this answers is not in the document]
Comment (anasereijo) [reply static/sass/_global-settings.scss]: I totally agree @Caleb-Ellis, thanks for checking. @steverydz can you please fix this?

**Comment by anasereijo** [comment]
@steverydz can you move the global nav to the bottom of the page? Please use the current design. @frankiegranato and I talked about it and for now we'll keep it that way.

can you please remove the padding on the left side when Enterprise is selected? Looks weird having just those few pixels as margin... the same for the right side
screen shot 2018-05-11 at 14 32 57

I can see a white horizontal line above containers.. can you please remove it?
screen shot 2018-05-11 at 14 37 36

align these 3 elements 
screen-shot-2018-05-11-at-14 38 58

can we have consistent vertical paddings?
screen-shot-2018-05-11-at-14 41 07

When the screen is larger than 320px can we please have back the 16px left and right margin on the links and distribute them horizontally?
screen shot 2018-05-11 at 14 44 37

File: static/sass/_global-settings.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (steverydz) [reply static/sass/_global-settings.scss]: Yep. I've no idea where that breakpoint came from?

**Comment by anasereijo** [comment]
@steverydz super fast... you're a star!

**Pull request merged**